pyfr/solvers/base/elements.py

```diff
@@ -47,6 +47,9 @@ def __init__(self, basiscls, eles, cfg):
         self.nfpts = basis.nfpts
         self.nfacefpts = basis.nfacefpts
 
+        # Compute metric (Smat, djac on pseudo grid points)
+        self._smat_base()
```

Comment:
Try to avoid generating in the constructor.  We are trying to transition away from this so that constructing Elements instances is a relatively cheap operation.